internal/querynodev2/segments/segment.go

```diff
@@ -73,39 +73,6 @@ type IndexedFieldInfo struct {
 	IndexInfo   *querypb.FieldIndexInfo
 }
 
-type Segment interface {
```

Comment:
Is this deleted on purpose?

Reply:
move to segment_interface.go,  make it easy to generate mock file